### ISSUE TITLE
Clean up build files

### ISSUE THREAD

**Issue by westin-m** [AzureAD/microsoft-identity-web#2295]
There are more yaml files than necessary for the build process. Clean these files up by un-nesting where appropriate and consolidating them.

**Comment by jmprieur** [comment]
Right, and some steps are run several times (Use net core sdk 2.x)
We should use dotnet test (not MSBuild) ... and understand what to do for the 2 OWIN projects. Use the new artifact folder
We could run binskip all at once, and probably the code signing?

**Comment by westin-m** [comment]
Here's (a diagram)[https://lucid.app/lucidchart/52e077ec-53b7-432d-8988-b1fbd9a5b91c/edit?viewport_loc=723%2C-436%2C2648%2C1410%2C0_0&invitationId=inv_24e84b8e-9705-4e2b-94bf-3baa2ae51f92] I whipped up to imagine what the final product could be. Let me know your thoughts.

**Comment by jennyf19** [comment]
> Here's [a diagram](https://lucid.app/lucidchart/52e077ec-53b7-432d-8988-b1fbd9a5b91c/edit?viewport_loc=723%2C-436%2C2648%2C1410%2C0_0&invitationId=inv_24e84b8e-9705-4e2b-94bf-3baa2ae51f92) I whipped up to imagine what the final product could be. Let me know your thoughts.

Looks great! We are hitting a lot of time-out issues now w/the new assembly too, so looking forward to getting some improvements in this area!

**Comment by jennyf19** [comment]
done.
